client/ayon_core/hosts/max/plugins/publish/collect_current_file.py

```diff
@@ -0,0 +1,23 @@
+import os
+import pyblish.api
+
+from pymxs import runtime as rt
+
+
+class CollectCurrentFile(pyblish.api.ContextPlugin):
+    """Inject the current working file."""
+
+    order = pyblish.api.CollectorOrder - 0.4
```

Comment:
This is not dependendent on anything before? I would move it to boundaries...
```suggestion
    order = pyblish.api.CollectorOrder - 0.5
```

Reply:
it isn't, so i would make a change on this